src/main/pages/che-7/overview/assembly_navigating-che-using-the-dashboard.adoc

```diff
@@ -15,9 +15,15 @@ summary:
 :context: navigating-che-using-the-dashboard
 
 The *Dashboard* is accessible on your cluster from a URL like `http://__<che-instance>__.__<IP-address>__.nip.io/dashboard/`.
-This section describes how to access this URL on Minishift, OpenShift Container Platform and OpenShift Dedicated.
+This section describes how to access this URL on
+ifeval::["{project-context}" == "che"]
+Minishift, 
```

Comment:
```suggestion
Minishift and
```

Reply:
I believe it is preferable to keep: `Minishift, OpenShift Container Platform and OpenShift Dedicated.`